frontend/src/pages/ListingsFeed.js

```diff
@@ -20,14 +21,37 @@ const Listings = () => {
         fetchListings();
     }, [dispatch]);
 
+    let listingsToShow;
+    switch (listingRequirement) {
+        case 'volunteering':
+            listingsToShow = listings.filter(listing => listing.requirement === 'Volunteering')
+            break;
+        case 'donation':
+            listingsToShow = listings.filter(listing => listing.requirement === 'Donation of Goods')
+            break;
+        default:
+            listingsToShow = listings;
+            break;
+    }
+
     return (
-        <div className="listings">
-            <h2>LISTINGS</h2>
-            {listings && listings.map((listing) => (
-                <ListingsFeed key={listing._id} listing={listing} />
-            ))}
-        </div>
-    )
+        <div class="dropdown">
```

Comment:
Just looking through this from my phone. The warning is because of this line here. JSX need to have the class as className.

Reply:
Nice one Ryan!